ui/component/headerCenterMenuButtons/view.jsx

```diff
@@ -0,0 +1,104 @@
+// @flow
+import 'scss/component/_header.scss';
+
+import { Link } from 'react-router-dom';
+import { Menu, MenuList, MenuButton, MenuItem, MenuLink } from '@reach/menu-button';
+import * as ICONS from 'constants/icons';
+import * as PAGES from 'constants/pages';
+import * as SETTINGS from 'constants/settings';
+import Icon from 'component/common/icon';
+import NotificationHeaderButton from 'component/headerNotificationButton';
+import React from 'react';
+import Tooltip from '@mui/material/Tooltip';
```

Comment:
New tooltip requires styling to match others + text too small

![image](https://user-images.githubusercontent.com/64950861/146296619-ed79b9c0-ba36-4652-802d-3c2f2d5590e7.png)

![image](https://user-images.githubusercontent.com/64950861/146296751-cc8ef4b2-6ade-4b65-8ef8-22040947792f.png)